lms/djangoapps/instructor/static/instructor/ProblemBrowser/data/store.js

```diff
@@ -0,0 +1,21 @@
+import {applyMiddleware, createStore} from 'redux';
+import thunkMiddleware from 'redux-thunk';
+
+import rootReducer from './reducers/index';
+
+const defaultState = {
+    blocks: [],
+    selectedBlock: null,
+    rootBlock: null,
+};
+
+const configureStore = initialState => createStore(
+    rootReducer,
+    initialState,
```

Comment:
This parameter is meant to be used for "universal apps" that hydrate the state server side and pass in some pre-initialized state to the client side, including some actions already applied.

Here, the reducers each define their own initial state (via the default argument `(state = blah)` in their function definition), so `defaultState` is redundant. You should just remove `defaultState`/`initialState` and let the reducers be the authoritative source of what the default state is.